proprietary/tokens/src/components/amsterdam/gap.tokens.json

```
@@ -0,0 +1,11 @@
+{
+  "amsterdam": {
+    "gap": {
+      "xs": { "value": "var(--amsterdam-spacing-fluid-xs)" },
```

Referring to other tokens is done like this:

```suggestion
      "xs": { "value": "{amsterdam.spacing.fluid.xs}" },
```

Same for the other gap tokens